clang/test/CIR/CodeGen/complex.cpp

```diff
@@ -176,3 +176,27 @@ void foo7() {
 // OGCG: store float %[[TMP_A]], ptr %[[C_REAL_PTR]], align 4
 // OGCG: store float 2.000000e+00, ptr %[[C_IMAG_PTR]], align 4
 
+void foo9(double r, double i) {
+  double _Complex c = __builtin_complex(r, i);
```

Comment:
Does `__builtin_complex` also accept integers? If so, add a test case for that.

I believe both arguments (or either) can also be an expression. So a test cases like this would be useful:

```
double _Complex c2 = __builtin_complex(a + b, a * 2.0);
double _Complex c3 = __builtin_complex(a + 1.0, 3.0);
```

Reply:
- `builtin_complex` accepts only floating points

- Both arguments can be expressed, but I need to implement the rest of the visitors in `ComplexExprEmitter`

Reply:
For arguments with expressions, I can update the tests when implementing new visitors